src/app/components/Application/Main.jsx

```diff
@@ -182,15 +182,15 @@ class Main extends React.Component {
     return updatedPicks;
   }
 
+
   render() {
     const { type } = this.props.picksData;
     const picksCount = this.getCount();
-
     return (
       <div className="nypl-row">
         <Sidebar
           filters={this.props.filters}
-          selectableFilters={this.state.selectableFilters}
```

Comment:
Since you're not using this anymore, you can remove this from the component.

Reply:
Removed `this.state.selectableFilters` from `Main`'s constructor.

Reply:
Right, it's not just there... Check the file again to see other instances of it in the state that are not being used.